src/controllers/accommodation.controller.js

```diff
@@ -0,0 +1,46 @@
+import AccommodationService from '../services/accommodation.service';
+import ResponseService from '../services/response.service';
+
+/**
+ *
+ *
+ * @class AccommodationController
+ */
+class AccommodationController {
+  /**
+           *
+           *
+           * @static
+           * @param {req} req
+           * @param {res} res
+           * @returns {response} @memberof AccommodationController
+           */
+  static async handleAccommodation(req, res) {
```

Comment:
`handleAccommodation` is too generic. Does this method creates, gets, deletes or updates an accommodation. That should reflect in the name.